tailwind.config.js

```diff
@@ -11,6 +11,10 @@ module.exports = {
     'src/**/*.tsx',
   ],
   theme: {
+    fontFamily: {
+      display: ['Poppins', 'sans-serif'],
+      text: ['"Work Sans"', 'sans-serif'], // consider 'body' instead?
```

Comment:
What do you have single quotes and double quotes around Work Sans?
Consider body for which part?

Reply:
It's explained here: https://tailwindcss.com/docs/font-family/#font-families

It's basically just to escape the space. I'll add that comment to the code itself.